kibana/Dockerfile

```diff
@@ -17,16 +19,12 @@ RUN   wget -q https://download.elastic.co/kibana/kibana/kibana-4.1.1-linux-x64.t
       tar -xzf kibana-4.1.1-linux-x64.tar.gz && \
       mv kibana-4.1.1-linux-x64/* /opt/app-root/src/ && \
       rm -rf kibana-4.1.1-linux-x64* && \
-      mkdir -m 755 /opt/origin-kibana && \
-      git clone --branch master --depth 1 https://github.com/openshift/origin-kibana.git /opt/origin-kibana && \
+      mkdir -p -m 755 /opt/origin-kibana/lib && \
+      cd /opt/origin-kibana/lib && \
+      wget -q ${AOP_KIBANA_PLUGIN_REPO}/releases/download/${AOP_KIBANA_PLUGIN_VER}/origin-kibana-${AOP_KIBANA_PLUGIN_VER}.tgz && \
+      tar -xvzf origin-kibana-${AOP_KIBANA_PLUGIN_VER}.tgz && \
```

Comment:
since we're suppressing wget's output, do we want to do the same for tar?  I feel like they should be consistent (one way or another)